clients/scheduler/src/components/time-slot-select/index.tsx

```diff
@@ -1,38 +1,63 @@
 import { h } from 'preact'
-import { Body, isSameDay, ParsedSlotsType, SlotType } from '@canvas-medical/embed-common'
+import {
+  Body,
+  isSameDay,
+  ParsedSlotsType,
+  SlotType,
+} from '@canvas-medical/embed-common'
 import { useState, useEffect, useMemo, useCallback } from 'preact/hooks'
 import { useAppContext } from '../../hooks'
 import { TimeSlotUi } from './ui'
 import { DateSelect } from '../date-select'
 
-const ONE_MINUTE_IN_MILLISECONDS = 60*1000
+const ONE_MINUTE_IN_MILLISECONDS = 60 * 1000
 
 export const TimeSlotSelect = () => {
-  const { fetchTimeSlots, date, setDate, daysToFetch, appointmentBufferInMintues } = useAppContext()
-  const [providerTimeSlots, setProviderTimeSlots] = useState<ParsedSlotsType[]>([])
-
-  const addTimeSlots = useCallback((newProviderTimeSlots: ParsedSlotsType[]) => {
-    const earliestAvailable = new Date();
-    earliestAvailable.setTime(earliestAvailable.getTime() + (appointmentBufferInMintues*ONE_MINUTE_IN_MILLISECONDS));
-
-    const mergedProviderAvailability = newProviderTimeSlots.map((newProvider) => {
-      const provider = providerTimeSlots.find((entry) => entry.providerId === newProvider.providerId)
-      const availableSlots = newProvider.providerSlots.filter(slot => new Date(slot.start) >= earliestAvailable)
-      const mergedSlots = provider ? [...provider.providerSlots, ...availableSlots] : availableSlots
-      return {providerId: newProvider.providerId, providerSlots: mergedSlots}
-    })
-
-    setProviderTimeSlots(mergedProviderAvailability)
-  }, [providerTimeSlots])
-
-  const {minDate, maxDate} = useMemo(() => {
+  const { fetchTimeSlots, date, setDate, appointmentBufferInMintues } =
```

Comment:
The only actual change to this file was removing `daysToFetch`, an unused variable. Other than that, this is just VScode formatter changes.